references/detection/train_tpu.py

```diff
@@ -229,14 +245,42 @@ def do_train(num_cores):
         lr_scheduler.step()
 
 
-
-
 def main():
     FLAGS = parse_flags()
     logger.info("FLAGS: {}".format(FLAGS))
 
-    do_prediction('/coco/train2017/000000566234.jpg', use_cpu=FLAGS.use_cpu)
-    #do_train(FLAGS.num_cores)
+    def mark_step():
```

Comment:
Just a suggestion, but do you think using this `mark_step()` may yield good info in eval?

Reply:
Hmm, the `evaluate()` function already calls `xm.mark_step()`. Or do you mean benchmarking it might be useful?

Reply:
Yeah the benchmarking version of `mark_step`. But an optional 😄